projects/backend/controllers/render.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+
+export const renderController = async (req: Request, res: Response) => {
+    const path = req.params.path
+    const renderingUrl = `${process.env.APPS_RENDERING_ROOT}${path}`
```

Comment:
I presume we need to set this url in aws as part of deployment?

Reply:
Yes this comes from `aws-stack.ts` and gets set in the cloudformation console